doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -412,10 +412,14 @@ Modem libraries
 
 * :ref:`nrf_modem_lib_readme`:
 
-   * Added a mention about enabling TF-M logging while using modem traces in the :ref:`modem_trace_module`.
-   * Updated by renaming ``lte_connectivity`` module to ``lte_net_if``.
-     All related Kconfig options have been renamed accordingly.
-   * Changed the default value of the :kconfig:option:`CONFIG_NRF_MODEM_LIB_NET_IF_AUTO_START`, :kconfig:option:`CONFIG_NRF_MODEM_LIB_NET_IF_AUTO_CONNECT`, and :kconfig:option:`CONFIG_NRF_MODEM_LIB_NET_IF_AUTO_DOWN` Kconfig options from enabled to disabled.
+  * Added:
+
+    * A mention about enabling TF-M logging while using modem traces in the :ref:`modem_trace_module`.
+    * The :kconfig:option:`CONFIG_NRF_MODEM_LIB_NET_IF_DOWN_DEFAULT_LTE_DISCONNECT` option, allowing the :c:enumerator:`~nrf_modem_lib_net_if_down_options.NRF_MODEM_LIB_NET_IF_DOWN_LTE_DISCONNECT` ``conn_mgr`` iface option to be selected at build time.
```

Comment:
```suggestion
    * The :kconfig:option:`CONFIG_NRF_MODEM_LIB_NET_IF_DOWN_DEFAULT_LTE_DISCONNECT` option, allowing the ``conn_mgr`` iface option of the :c:enumerator:`NRF_MODEM_LIB_NET_IF_DOWN` enumerator to be selected at build time.
```
I may have messed up this totally. However, this needs a bit closer study. The enumerator was not rendered properly.

Reply:
Hmm, I've had this exact syntax work over on the Zephyr documentation:

https://docs.zephyrproject.org/latest/connectivity/networking/conn_mgr/main.html#connection-persistence

```
Each iface also has a connection persistence setting that you can enable or disable by setting the :c:enumerator:`~conn_mgr_if_flag.CONN_MGR_IF_PERSISTENT` flag with :c:func:`conn_mgr_binding_set_flag`.
```

I'll dig into it

Reply:
Ah, I spotted the problem I think. The `nrf_modem_lib_net_if_down_options` struct is inside a preprocessor guard that's missing `|| defined(__DOXYGEN__)`

Reply:
Yep, appears to work now